descarga_datos/tests/test_Analysis.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import json
+
+from .. import Analysis
+
+TEXTO_ANALYSIS = """[
+    {
+        "name": "Cantidad de individuos por transecto con captura y recaptura",
+        "description": "Cantidad de individuos por transecto con captura y recaptura en Isla Cedros",
+        "docker_parent_image": "islasgeci/extension:d25d",
+        "report": "",
+        "results": [
+        ],
+        "scripts": [
+        ],
+        "data": [
+            {
+                "source": "datapackage",
+                "path": "roedores_capturarecaptura_cedros",
+                "filename": "roedores_capturarecaptura_cedros.csv",
+                "version": "d2ca5a04850b",
+                "type": "datapackage"
+            }
+        ],
+        "requirements": []
+    }
+]"""
+
+
+class TestAnalysis(unittest.TestCase):
+
+    def setUp(self):
+        diccionario_analysis = json.loads(TEXTO_ANALYSIS)
+        analisis = diccionario_analysis[0]
+        self.analisis = Analysis(**analisis)
+
+    def test_datafile_name(self):
+        self.assertEqual(
+            self.analisis._data[0].filename, "roedores_capturarecaptura_cedros.csv")
+
+    def test_analysis_name(self):
+        self.assertEqual(
+            self.analisis.name, "Cantidad de individuos por transecto con captura y recaptura")
+
+    def test_depends_of_file(self):
+        self.assertTrue(self.analisis.depends_on_file(
+            "roedores_capturarecaptura_cedros.csv"))
+
+    def test_get_utl_to_file(self):
```

Comment:
¿Hay aquí un _typo_?

```suggestion
    def test_get_url_to_file(self):
```